src/legacy/Routes.tsx

```diff
@@ -109,11 +109,11 @@ class RouteApp extends React.Component<any, any> {
     this.rpc = rpc
 
     // Auto refresh every 10s
-    this.rpcRefreshIntervalId = window.setInterval(() => {
-      if (this.state.rpcConfig.username) {
-        rpc.refresh()
-      }
-    }, 10000)
+    // this.rpcRefreshIntervalId = window.setInterval(() => {
+    //   if (this.state.rpcConfig.username) {
+    //     rpc.refresh()
+    //   }
+    // }, 10000)
```

Comment:
why did we need to remove this? Is this remove for the development mode? If yes, can we use `if (!app.isPackaged) {` ?

please refer to: https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/main.ts Line 87